ioctl/client.go

```diff
@@ -122,16 +131,21 @@ func EnableCryptoSm2() Option {
 }
 
 // NewClient creates a new ioctl client
-func NewClient(cfg config.Config, configFilePath string, opts ...Option) Client {
+func NewClient(cfg config.Config, configFilePath string, opts ...Option) (Client, error) {
+	xrc20ABI, err := abi.JSON(strings.NewReader(_xrc20ABI))
+	if err != nil {
+		return nil, errors.Wrap(err, "cannot get abi JSON data")
+	}
```

Comment:
we may put it into `init`, and panic if any error

Reply:
fixed